.gitignore

```diff
@@ -0,0 +1,2 @@
+# Mari's VS code folder.
```

Comment:
Is this another .gitignore? If so, why the server folder being ignored?

Reply:
Done.